js/views/userPageVw.js

```diff
@@ -303,10 +303,19 @@ UserPageVw = pageVw.extend({
             model.set('headerURL', self.options.userModel.get('serverUrl') + "get_image?hash=" + model.get('profile').header_hash + "&guid=" + self.pageID);
             model.set('avatarURL', self.options.userModel.get('serverUrl') + "get_image?hash=" + model.get('profile').avatar_hash + "&guid=" + self.pageID);
           }
+
           // Cache user avatar in localStorage
           var profile = model.toJSON().profile;
           window.localStorage.setItem("avatar_" + self.pageID, profile.avatar_hash);
 
+          if (response.profile.website) {
+            var website = profile.website;
+            var parse_url_regex = /^(?:([A-Za-z]+):)?(\/{0,3})([0-9.\-A-Za-z]+)(?::(\d+))?(?:\/([^?#]*))?(?:\?([^#]*))?(?:#(.*))?$/;
```

Comment:
Do you need all of the regex forumula, or would 
var parse_url_regex = /^(?:([A-Za-z]+):)?(\/{0,3})([0-9.\-A-Za-z]+)/;
be enough?

You might be able to make this more efficient with a split on :// and :, ? or /, and take the 2nd result.

Another option would be to use Chromium's built in parsing. 

var parser = document.createElement('a');
parser.href = profile.website;
profile.short_website = parser.hostname;